l10n_ec_account_edi/models/account_edi_document.py

```diff
@@ -724,3 +724,15 @@ def l10n_ec_send_mail_to_partner(self):
             ]
         )
         account_moves_with_final_consumer.write({"is_move_sent": True})
+
+    def action_unauthorized_documents_notification(self):
+        documents = self.env["res.company"].search([])
+
+        email_template = self.env.ref(
+            "l10n_ec_account_edi.email_template_unauthorized_notify", False
+        )
+
+        for document in documents:
+            email_template.send_mail(document.id, force_send=True)
```

Comment:
Here you pass document belonging to `res.company`, but it should be the same model used in the email template. Over there, you use `account.edi.document`. Please change the `model_id` in the email template to `base.model_res_company`.